lib/sneakers/configuration.rb

```diff
@@ -1,4 +1,5 @@
 require 'forwardable'
+require 'active_support/core_ext/hash/deep_merge'
```

Comment:
I'm uncertain we can transparently support `active_support` - it is not in the dependencies for Sneakers, and it might feel a bit heavy as a dependencies for other users of Sneakers. Perhaps there is a way to extract deep merge or use an alternative one?